stack-snapshot.yaml

```diff
@@ -13,10 +13,11 @@ packages:
   - github: digital-asset/hlint
     commit: "99999eee1c220396e1545788ef22ca3c204908a9"
     sha256: "52c8a636f29ecca4cdd0fa9ed094bae3c0115cb10a7c6b3e6087361a8b4ecffb"
+  - mod-0.1.2.2
+  - optparse-applicative-0.17.1.0
```

Comment:
previously we were on `optparse-applicative-0.16.1.0` since that's the version in stackage `nightly-2022-01-26`, but the completion scripts were only exposed in `0.17.1.0` [[optparse-applicative changelog]](https://hackage.haskell.org/package/optparse-applicative-0.18.1.0/changelog)

Reply:
Did said scripts change between those versions?

Reply:
also maybe we should read through the optparse-applicative code to ensure the scripts are mostly hardcoded, only using the users path

Reply:
vdiff checks out ;)
https://hackage.haskell.org/package/optparse-applicative-0.16.1.0/docs/src/Options.Applicative.BashCompletion.html#bashCompletionScript
https://hackage.haskell.org/package/optparse-applicative-0.17.1.0/docs/src/Options.Applicative.BashCompletion.html#bashCompletionScript

Reply:
(the only difference (other than the fact that these weren't exported) is that the functions in `0.16.1.0` returned a list of strings, but in `0.17.1.0` they are unlined)